web/packages/teleport/src/User/UserContext.tsx

```diff
@@ -139,6 +187,10 @@ export function UserContextProvider(props: PropsWithChildren<unknown>) {
     run(loadUserPreferences);
   }, []);
 
+  useEffect(() => {
+    loadClusterPreferences();
```

Comment:
This should be wrapped in an attempt, similarly to the regular request for preferences. When the request is processing, we should show a loading indicator.